frontend/packages/signless-transactions/src/hooks/use-create-session.ts

```diff
@@ -59,19 +59,26 @@ function useCreateSession(programId: HexString, metadata: ProgramMetadata | unde
     return api.message.send({ destination, payload, gasLimit }, metadata);
   };
 
+  const getDurationBlocks = (durationMS: number) => {
+    const secondsPerBLock = 3;
+
+    return Math.round(durationMS / 1000 / secondsPerBLock);
```

Comment:
```suggestion
    const blockTimeMs = api?.consts.babe.expectedBlockTime.toNumber();

    return durationMS / blockTimeMs;
```

need to use chain constant